### PR TITLE
Removed generators in test_calling_convention_analysis.py

### PR DESCRIPTION
**Pull request by mohitrpatil**

### CODE REVIEW

**Review by rhelmot — APPROVED** [review]

File: tests/test_calling_convention_analysis.py [hunk 1]
```diff
@@ -66,7 +66,7 @@ def test_fauxware(self):
         }
 
         for arch, lst in args.items():
-            yield self.run_fauxware, arch, lst
+            self._run_fauxware, arch, lst
```
Comment (ltfish) [inline tests/test_calling_convention_analysis.py:69]: Seems like a useless statement to me.

File: tests/test_calling_convention_analysis.py [hunk 2]
```diff
@@ -51,32 +51,35 @@ def run_cgc(self, binary_name):
...
-
+    def test_fauxware_i386(self):
         amd64 = archinfo.arch_from_id("amd64")
 
         args = {
             'i386': [
                 ('authenticate', SimCCCdecl(archinfo.arch_from_id('i386'), )),
             ],
+        }
+        for arch , lst in args.items():
```
Comment (twizmwazin) [inline tests/test_calling_convention_analysis.py:62]: Since the dict only has a single item in it, let's remove this and pass the arguments directly.

**Review by twizmwazin — CHANGES_REQUESTED** [review]

**Review by twizmwazin — COMMENTED** [review]
Please fix the lint and I'll hit merge! Looks like you might have removed the `@requires_binaries_private` and forgot to re-add it.

**Comment by twizmwazin** [comment]
Thanks!

**Pull request merged**